metadata/edgex-device-camera.md

```diff
@@ -1,14 +1,15 @@
 EdgeX Device Camera
 ---
 The EdgeX Device Camera is a device service for connecting ONVIF-compliant cameras to EdgeX.
+The services enabled EdgeX to read from and control the camera.
```

Comment:
It is important to say the service adds something to EdgeX, rather than being a standalone component created by EdgeX.

I replaced with:
> The EdgeX Device Camera is a device service for communicating with ONVIF-compliant cameras via EdgeX.
It can be used to get and set camera attributes and retrieve snapshot images.

I think including what is supported in sufficient.